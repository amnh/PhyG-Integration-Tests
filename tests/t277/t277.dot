digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=0.0];
    16 -> 30 [label=0.0];
    17 -> 28 [label=338.0];
    17 -> 29 [label=73.5];
    18 -> 4 [label=603.0];
    18 -> 19 [label=724.6];
    19 -> 7 [label=1054.1];
    19 -> 15 [label=3273.1];
    20 -> 8 [label=5895.9];
    20 -> 12 [label=1168.1];
    21 -> 1 [label=2455.2];
    21 -> 25 [label=275.0];
    22 -> 2 [label=1565.0];
    22 -> 17 [label=72.0];
    23 -> 5 [label=1161.9];
    23 -> 22 [label=159.6];
    24 -> 6 [label=973.0];
    24 -> 23 [label=367.0];
    25 -> 13 [label=1958.0];
    25 -> 24 [label=1138.9];
    26 -> 9 [label=927.4];
    26 -> 14 [label=1413.2];
    27 -> 3 [label=3981.6];
    27 -> 20 [label=411.0];
    28 -> 11 [label=421.6];
    28 -> 18 [label=82.0];
    29 -> 26 [label=203.0];
    29 -> 27 [label=170.1];
    30 -> 10 [label=5271.8];
    30 -> 21 [label=490.9];
}
//48504.4
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=0.0];
    16 -> 30 [label=0.0];
    17 -> 28 [label=338.0];
    17 -> 29 [label=73.5];
    18 -> 4 [label=603.0];
    18 -> 19 [label=724.6];
    19 -> 7 [label=1054.1];
    19 -> 15 [label=3273.1];
    20 -> 8 [label=5895.9];
    20 -> 12 [label=1168.1];
    21 -> 1 [label=2455.2];
    21 -> 25 [label=275.0];
    22 -> 2 [label=1565.0];
    22 -> 17 [label=72.0];
    23 -> 5 [label=1161.9];
    23 -> 22 [label=159.6];
    24 -> 6 [label=973.0];
    24 -> 23 [label=367.0];
    25 -> 13 [label=1958.0];
    25 -> 24 [label=1138.9];
    26 -> 9 [label=927.4];
    26 -> 14 [label=1413.2];
    27 -> 3 [label=3981.6];
    27 -> 20 [label=411.0];
    28 -> 11 [label=421.6];
    28 -> 18 [label=82.0];
    29 -> 26 [label=203.0];
    29 -> 27 [label=170.1];
    30 -> 10 [label=5271.8];
    30 -> 21 [label=490.9];
}
//48504.4
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=0.0];
    16 -> 30 [label=0.0];
    17 -> 28 [label=191.0];
    17 -> 29 [label=95.4];
    18 -> 4 [label=755.0];
    18 -> 7 [label=1316.7];
    19 -> 3 [label=2673.5];
    19 -> 15 [label=2811.0];
    20 -> 8 [label=5174.0];
    20 -> 12 [label=1100.0];
    21 -> 1 [label=2428.2];
    21 -> 25 [label=275.0];
    22 -> 2 [label=1563.0];
    22 -> 17 [label=63.0];
    23 -> 5 [label=1148.9];
    23 -> 22 [label=186.6];
    24 -> 6 [label=961.0];
    24 -> 23 [label=376.0];
    25 -> 13 [label=1978.0];
    25 -> 24 [label=1152.9];
    26 -> 9 [label=936.3];
    26 -> 14 [label=1369.3];
    27 -> 19 [label=574.2];
    27 -> 20 [label=240.0];
    28 -> 11 [label=360.6];
    28 -> 18 [label=97.0];
    29 -> 26 [label=197.0];
    29 -> 27 [label=149.1];
    30 -> 10 [label=5246.8];
    30 -> 21 [label=486.9];
}
//47887.3
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=0.0];
    16 -> 30 [label=0.0];
    17 -> 28 [label=191.0];
    17 -> 29 [label=95.4];
    18 -> 4 [label=755.0];
    18 -> 7 [label=1316.7];
    19 -> 3 [label=2673.5];
    19 -> 15 [label=2811.0];
    20 -> 8 [label=5174.0];
    20 -> 12 [label=1100.0];
    21 -> 1 [label=2428.2];
    21 -> 25 [label=275.0];
    22 -> 2 [label=1563.0];
    22 -> 17 [label=63.0];
    23 -> 5 [label=1148.9];
    23 -> 22 [label=186.6];
    24 -> 6 [label=961.0];
    24 -> 23 [label=376.0];
    25 -> 13 [label=1978.0];
    25 -> 24 [label=1152.9];
    26 -> 9 [label=936.3];
    26 -> 14 [label=1369.3];
    27 -> 19 [label=574.2];
    27 -> 20 [label=240.0];
    28 -> 11 [label=360.6];
    28 -> 18 [label=97.0];
    29 -> 26 [label=197.0];
    29 -> 27 [label=149.1];
    30 -> 10 [label=5246.8];
    30 -> 21 [label=486.9];
}
//47887.3
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    16 -> 0 [label=0.0];
    16 -> 34 [label=0.0];
    17 -> 32 [label=200.0];
    17 -> 33 [label=96.0];
    18 -> 4 [label=670.0];
    18 -> 19 [label=975.7];
    19 -> 20 [label=257.0];
    19 -> 28 [label=283.0];
    20 -> 7 [label=248.0];
    20 -> 29 [label=0.0];
    21 -> 3 [label=2668.5];
    21 -> 29 [label=401.0];
    22 -> 8 [label=5177.0];
    22 -> 12 [label=1016.0];
    23 -> 1 [label=2448.2];
    23 -> 27 [label=270.0];
    24 -> 2 [label=1542.0];
    24 -> 17 [label=52.0];
    25 -> 5 [label=1015.9];
    25 -> 24 [label=179.2];
    26 -> 6 [label=981.0];
    26 -> 25 [label=381.0];
    27 -> 13 [label=1975.0];
    27 -> 26 [label=1138.9];
    28 -> 14 [label=651.0];
    29 -> 15 [label=25.0];
    30 -> 9 [label=514.0];
    30 -> 28 [label=0.0];
    31 -> 21 [label=476.2];
    31 -> 22 [label=310.0];
    32 -> 11 [label=341.2];
    32 -> 18 [label=77.4];
    33 -> 30 [label=254.5];
    33 -> 31 [label=139.9];
    34 -> 10 [label=5273.8];
    34 -> 23 [label=490.9];
}
//47358.95
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=0.0];
    16 -> 30 [label=0.0];
    17 -> 28 [label=199.0];
    17 -> 29 [label=96.0];
    18 -> 4 [label=670.0];
    18 -> 19 [label=975.7];
    19 -> 7 [label=510.0];
    19 -> 14 [label=933.0];
    20 -> 3 [label=2667.5];
    20 -> 15 [label=2791.0];
    21 -> 8 [label=5178.0];
    21 -> 12 [label=1016.0];
    22 -> 1 [label=2447.2];
    22 -> 26 [label=270.0];
    23 -> 2 [label=1543.0];
    23 -> 17 [label=52.0];
    24 -> 5 [label=1017.9];
    24 -> 23 [label=178.2];
    25 -> 6 [label=982.0];
    25 -> 24 [label=383.0];
    26 -> 13 [label=1974.0];
    26 -> 25 [label=1135.9];
    27 -> 20 [label=476.2];
    27 -> 21 [label=310.0];
    28 -> 11 [label=341.2];
    28 -> 18 [label=77.4];
    29 -> 9 [label=768.5];
    29 -> 27 [label=139.9];
    30 -> 10 [label=5274.8];
    30 -> 22 [label=490.9];
}
//47350.2
