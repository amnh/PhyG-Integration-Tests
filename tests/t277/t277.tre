(AlbanianTosk:0.0,(KolschColognian:5271.8,(AncientIonicGreek:2455.2,(SanskritVedic:1958.0,(EasternFrisianSaterland:973.0,(Danish:1161.9,(ArmenianWestern:1565.0,((Manx:421.6,(Cornish:603.0,(Faroese:1054.1,Yiddish:3273.1)HTU19:724.6)HTU18:82.0)HTU28:338.0,((Irish:927.4,Scots:1413.2)HTU26:203.0,(Assamese:3981.6,(Icelandic:5895.9,OldPrussian:1168.1)HTU20:411.0)HTU27:170.1)HTU29:73.5)HTU17:72.0)HTU22:159.6)HTU23:367.0)HTU24:1138.9)HTU25:275.0)HTU21:490.9)HTU30:0.0)HTU16[48504.4];
(AlbanianTosk:0.0,(KolschColognian:5271.8,(AncientIonicGreek:2455.2,(SanskritVedic:1958.0,(EasternFrisianSaterland:973.0,(Danish:1161.9,(ArmenianWestern:1565.0,((Manx:421.6,(Cornish:603.0,(Faroese:1054.1,Yiddish:3273.1)HTU19:724.6)HTU18:82.0)HTU28:338.0,((Irish:927.4,Scots:1413.2)HTU26:203.0,(Assamese:3981.6,(Icelandic:5895.9,OldPrussian:1168.1)HTU20:411.0)HTU27:170.1)HTU29:73.5)HTU17:72.0)HTU22:159.6)HTU23:367.0)HTU24:1138.9)HTU25:275.0)HTU21:490.9)HTU30:0.0)HTU16[48504.4];
(AlbanianTosk:0.0,(KolschColognian:5246.8,(AncientIonicGreek:2428.2,(SanskritVedic:1978.0,(EasternFrisianSaterland:961.0,(Danish:1148.9,(ArmenianWestern:1563.0,((Manx:360.6,(Cornish:755.0,Faroese:1316.7)HTU18:97.0)HTU28:191.0,((Irish:936.3,Scots:1369.3)HTU26:197.0,((Assamese:2673.5,Yiddish:2811.0)HTU19:574.2,(Icelandic:5174.0,OldPrussian:1100.0)HTU20:240.0)HTU27:149.1)HTU29:95.4)HTU17:63.0)HTU22:186.6)HTU23:376.0)HTU24:1152.9)HTU25:275.0)HTU21:486.9)HTU30:0.0)HTU16[47887.3];
(AlbanianTosk:0.0,(KolschColognian:5246.8,(AncientIonicGreek:2428.2,(SanskritVedic:1978.0,(EasternFrisianSaterland:961.0,(Danish:1148.9,(ArmenianWestern:1563.0,((Manx:360.6,(Cornish:755.0,Faroese:1316.7)HTU18:97.0)HTU28:191.0,((Irish:936.3,Scots:1369.3)HTU26:197.0,((Assamese:2673.5,Yiddish:2811.0)HTU19:574.2,(Icelandic:5174.0,OldPrussian:1100.0)HTU20:240.0)HTU27:149.1)HTU29:95.4)HTU17:63.0)HTU22:186.6)HTU23:376.0)HTU24:1152.9)HTU25:275.0)HTU21:486.9)HTU30:0.0)HTU16[47887.3];
(AlbanianTosk:0.0,(KolschColognian:5273.8,(AncientIonicGreek:2448.2,(SanskritVedic:1975.0,(EasternFrisianSaterland:981.0,(Danish:1015.9,(ArmenianWestern:1542.0,((Manx:341.2,(Cornish:670.0,((Faroese:248.0,(Yiddish:0.0)#HTU29:0.0)HTU20:257.0,(Scots:283.0)#HTU28:283.0)HTU19:975.7)HTU18:77.4)HTU32:200.0,((Irish:514.0,#HTU28:0.0)HTU30:254.5,((Assamese:2668.5,#HTU29:401.0)HTU21:476.2,(Icelandic:5177.0,OldPrussian:1016.0)HTU22:310.0)HTU31:139.9)HTU33:96.0)HTU17:52.0)HTU24:179.2)HTU25:381.0)HTU26:1138.9)HTU27:270.0)HTU23:490.9)HTU34:0.0)HTU16[47358.95];
(AlbanianTosk:0.0,(KolschColognian:5274.8,(AncientIonicGreek:2447.2,(SanskritVedic:1974.0,(EasternFrisianSaterland:982.0,(Danish:1017.9,(ArmenianWestern:1543.0,((Manx:341.2,(Cornish:670.0,(Faroese:510.0,Scots:933.0)HTU19:975.7)HTU18:77.4)HTU28:199.0,(Irish:768.5,((Assamese:2667.5,Yiddish:2791.0)HTU20:476.2,(Icelandic:5178.0,OldPrussian:1016.0)HTU21:310.0)HTU27:139.9)HTU29:96.0)HTU17:52.0)HTU23:178.2)HTU24:383.0)HTU25:1135.9)HTU26:270.0)HTU22:490.9)HTU30:0.0)HTU16[47350.2];
