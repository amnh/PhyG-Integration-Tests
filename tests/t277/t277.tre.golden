(AlbanianTosk:396.55,(KolschColognian:6186.4,(AncientIonicGreek:3471.3,(SanskritVedic:3318.6,(EasternFrisianSaterland:2403.1,(Danish:2414.95,(ArmenianWestern:2831.45,((Manx:1329.15,(Cornish:1487.6,(Faroese:1776.1999999999998,Yiddish:4099.65)HTU19:1245.7)HTU18:423.1)HTU28:1594.1,((Irish:1774.45,Scots:2108.8)HTU26:1679.1,(Assamese:5870.200000000001,(Icelandic:6501.45,OldPrussian:1677.65)HTU20:2352.1)HTU27:1450.6499999999999)HTU29:1205.6)HTU17:899.1)HTU22:897.65)HTU23:1139.1)HTU24:2292.4)HTU25:1421.6)HTU21:1195.95)HTU30:425.05)HTU16[48504.4];
(AlbanianTosk:396.55,(KolschColognian:6186.4,(AncientIonicGreek:3471.3,(SanskritVedic:3318.6,(EasternFrisianSaterland:2403.1,(Danish:2414.95,(ArmenianWestern:2831.45,((Manx:1329.15,(Cornish:1487.6,(Faroese:1776.1999999999998,Yiddish:4099.65)HTU19:1245.7)HTU18:423.1)HTU28:1594.1,((Irish:1774.45,Scots:2108.8)HTU26:1679.1,(Assamese:5870.200000000001,(Icelandic:6501.45,OldPrussian:1677.65)HTU20:2352.1)HTU27:1450.6499999999999)HTU29:1205.6)HTU17:899.1)HTU22:897.65)HTU23:1139.1)HTU24:2292.4)HTU25:1421.6)HTU21:1195.95)HTU30:425.05)HTU16[48504.4];
(AlbanianTosk:398.05,(KolschColognian:6197.4,(AncientIonicGreek:3471.3,(SanskritVedic:3336.6,(EasternFrisianSaterland:2383.6,(Danish:2378.45,(ArmenianWestern:2784.45,((Manx:1289.15,(Cornish:1667.6,Faroese:2057.8)HTU18:492.1)HTU28:1587.6,((Irish:1750.35,Scots:2032.9)HTU26:1707.6,((Assamese:4828.6,Yiddish:5268.1)HTU19:2554.2999999999997,(Icelandic:6172.55,OldPrussian:2173.05)HTU20:2630.6)HTU27:1880.1499999999999)HTU29:1336.0)HTU17:1022.6)HTU22:949.6500000000001)HTU23:1190.6)HTU24:2304.9)HTU25:1432.1)HTU21:1175.95)HTU30:326.05)HTU16[47887.3];
(AlbanianTosk:398.05,(KolschColognian:6197.4,(AncientIonicGreek:3471.3,(SanskritVedic:3336.6,(EasternFrisianSaterland:2383.6,(Danish:2378.45,(ArmenianWestern:2784.45,((Manx:1289.15,(Cornish:1667.6,Faroese:2057.8)HTU18:492.1)HTU28:1587.6,((Irish:1750.35,Scots:2032.9)HTU26:1707.6,((Assamese:4828.6,Yiddish:5268.1)HTU19:2554.2999999999997,(Icelandic:6172.55,OldPrussian:2173.05)HTU20:2630.6)HTU27:1880.1499999999999)HTU29:1336.0)HTU17:1022.6)HTU22:949.6500000000001)HTU23:1190.6)HTU24:2304.9)HTU25:1432.1)HTU21:1175.95)HTU30:326.05)HTU16[47887.3];
(AlbanianTosk:399.55,(KolschColognian:6207.9,(AncientIonicGreek:3480.8,(SanskritVedic:3355.6,(EasternFrisianSaterland:2423.6,(Danish:2355.95,(ArmenianWestern:2887.45,((Manx:1402.8000000000002,(Cornish:1611.1,((Faroese:792.6,(Yiddish:1.0)#HTU29:1.0)HTU20:418.1,(Scots:488.6)#HTU28:488.6)HTU19:1344.3000000000002)HTU18:509.45)HTU32:1486.1,((Irish:2240.1,#HTU28:0.0)HTU30:1283.6,((Assamese:4722.1,#HTU29:3801.6)HTU21:2305.3,(Icelandic:6261.05,OldPrussian:2198.05)HTU22:2462.1)HTU31:1711.9)HTU33:1163.1)HTU17:926.6)HTU24:993.8000000000001)HTU25:1215.1)HTU26:2288.9)HTU27:1404.6)HTU23:1194.95)HTU34:428.05)HTU16[47358.95];
(AlbanianTosk:397.55,(KolschColognian:6208.4,(AncientIonicGreek:3478.3,(SanskritVedic:3355.1,(EasternFrisianSaterland:2424.1,(Danish:2357.45,(ArmenianWestern:2888.45,((Manx:1403.3000000000002,(Cornish:1610.6,(Faroese:1129.1,Scots:1558.1)HTU19:1344.3000000000002)HTU18:509.95)HTU28:1485.1,(Irish:2758.1,((Assamese:4721.1,Yiddish:5133.6)HTU20:2308.3,(Icelandic:6261.55,OldPrussian:2197.55)HTU21:2461.1)HTU27:1712.4)HTU29:1162.1)HTU17:928.6)HTU23:991.8000000000001)HTU24:1216.6)HTU25:2287.4)HTU26:1405.1)HTU22:1193.95)HTU30:425.55)HTU16[47350.2];
